{
    "ImageMagick/ImageMagick6": [
        {
            "cve_name": "CVE-2017-6499",
            "patch_url": "https://github.com/ImageMagick/ImageMagick6/commit/3358f060fc182551822576b2c0a8850faab5d543.patch",
            "reaching_path": "https://security-tracker.debian.org/tracker/CVE-2017-6499",
            "repository_url": "ImageMagick/ImageMagick6",
            "owner": "ImageMagick",
            "repo": "ImageMagick6",
            "base_hash": "85cc3446bf207ca5460ebbbe18b81493739eda1c",
            "head_hash": "3358f060fc182551822576b2c0a8850faab5d543",
            "function_names": [
                "Magick::throwException"
            ],
            "old_files": [
                {
                    "filename": "Magick++/lib/Exception.cpp",
                    "lines": [
                        855,
                        858,
                        859,
                        860
                    ]
                }
            ],
            "new_files": [
                {
                    "filename": "Magick++/lib/Exception.cpp",
                    "lines": [
                        855,
                        856,
                        857,
                        858,
                        861,
                        862,
                        864,
                        865,
                        866
                    ]
                }
            ],
            "affected_version": [
                "6.9.7"
            ],
            "binary_version": [
                "6.9.7-0"
            ]
        }
    ]
}